static/js/vue-cdr-access/src/mixins/routeUtils.js

```diff
@@ -125,9 +125,13 @@ export default {
         /**
          * Returns a list of query parameters from the current URLs minus the specified set of parameters
          * @param param_names names of query parameters to remove
+         * @param reset_start_row Start row should be reset to 0 for searching, pagination purposes
          * @returns {LocationQuery} list of query parameters with specified parameters removed
          */
-        removeQueryParameters(param_names) {
+        removeQueryParameters(param_names, reset_start_row = false) {
```

Comment:
I don't think `reset_start_row` is ever used now, so can remove it and the block that goes with it

Reply:
Whoops, thought I'd removed this.